sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -757,44 +757,45 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
   def entryPointsTask(stage: TaskKey[Attributed[File]]): Def.Initialize[Task[Seq[(String, File)]]] =
     Def.taskDyn {
+      val s = streams.value
```

Comment:
Yes that's the idea. 

See my earlier comment on grabbing the linkingUnit. Basically not possible in any clean way.